standard/statements.md

```diff
@@ -1179,26 +1179,13 @@ It is a compile-time error for a `break`, `continue`, or `goto` statement to tra
 
 It is a compile-time error for a `return` statement to occur in a `finally` block.
 
-A `try` statement is executed as follows:
-
-- Control is transferred to the `try` block.
-- When and if control reaches the end point of the `try` block:
-  - If the `try` statement has a `finally` block, the `finally` block is executed.
-  - Control is transferred to the end point of the `try` statement.
-- If an exception is propagated to the `try` statement during execution of the `try` block:
-  - The `catch` clauses, if any, are examined in order of appearance to locate a suitable handler for the exception. Each `catch` clause that does not specify a type, or specifies the exception type or a base type of the exception type, is considered a match. If a matching `catch` clause is located:
-    - If the matching `catch` clause declares an exception variable, the exception object is assigned to the exception variable.
-    - If the `catch` clause declares an exception filter, the filter is evaluated. If it evaluates to `false`, the search continues through any subsequent `catch` clauses for a suitable handler.    
-    - Otherwise control is transferred to the corresponding `catch` block.
-    - When and if control reaches the end point of the `catch` block:
-      - If the `try` statement has a `finally` block, the `finally` block is executed.
-      - Control is transferred to the end point of the `try` statement.
-    - If an exception is propagated to the `try` statement during execution of the `catch` block:
-      - If the `try` statement has a `finally` block, the `finally` block is executed.
-      - The exception is propagated to the next enclosing `try` statement.
-- If the `try` statement has no `catch` clauses or if no `catch` clause matches the exception:
-  - If the `try` statement has a `finally` block, the `finally` block is executed.
-  - The exception is propagated to the next enclosing `try` statement.
+When execution reaches a`try` statement, control is transferred to the `try` block. If control reaches the end point of the `try` block without an exception being propagated, control is transferred to the `finally` block if one exists. If no `finally` block exists, control is transferred to the end point of the `try` statement.
```

Comment:
Original:

> - When and if control reaches the end point of the `try` block:
>   - If the `try` statement has a `finally` block, the `finally` block is executed.
>   - Control is transferred to the end point of the `try` statement.

This states the finally block is conditionally executed and then control is transferred to the end of the try statement.

Replacement:

> If control reaches the end point of the `try` block without an exception being propagated, control is transferred to the `finally` block if one exists. If no `finally` block exists, control is transferred to the end point of the `try` statement.

This says control is transferred to the finally block if it exists *otherwise* to the end of the try statement.

Hasn't this lost what happens when a finally block finishes executing?

Reply:
Thanks @Nigel-Ecma 

I addressed this in the paragraph on 1190. That defines how the statements in a `finally` block execute. I think the prose worked better.

Reply:
The original talks about executing the `finally` and then transferring control to the endpoint of the `try` statement. Doesn’t 1190 cover the executing part and not the transfer to the endpoint bit?

Reply:
@BillWagner – So you have, my bad I was looking at the wrong commit, apologies.